File: java/button with alert ionic/buttonwithalert/app/src/main/java/com/gradproject/buttonwithalert/MainActivity.java
package com.gradproject.buttonwithalert;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.content.DialogInterface;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button alertdialogbutton;
    private AlertDialog.Builder alertDialog;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        alertdialogbutton = findViewById(R.id.button);
        alertdialogbutton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                alertDialog = new AlertDialog.Builder(MainActivity.this);
                alertDialog.setTitle("localhost:8101 says");
                alertDialog.setMessage("Button Clicked!");
                alertDialog.setPositiveButton("OK", new DialogInterface.OnClickListener() {
                    @Override
                    public void onClick(DialogInterface dialog, int which) {
                        dialog.cancel();
                    }
                });
                AlertDialog dialog = alertDialog.create();
                dialog.show();
                            }
        });
    }
}

File: Kotlin/Hello world flutter/app/src/main/java/com/badawy5/formflutter/MainActivity.kt
package com.badawy5.formflutter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AlertDialog

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button1.setOnClickListener({
            val alertDialogBuilder = AlertDialog.Builder(this).setTitle("localhost:8101 says")
                .setMessage("Button Clicked!").setNegativeButton("Ok") { dialogInterface, which ->
                Toast.makeText(applicationContext, "clicked ok", Toast.LENGTH_LONG).show()

            }.show()
        })

    }
}